packages/mask/src/plugins/Predict/Azuro/views/EventsView.tsx

```diff
@@ -0,0 +1,171 @@
+        width: 150,
+    },
+    field: {
+        width: 100,
+    },
+    refreshButton: {
+        borderRadius: '15px',
+        border: `solid 1px ${theme.palette.divider}`,
+    },
+}))
+
+export function EventsView(): JSX.Element {
+    const { t: tr } = useBaseI18N()
+    const t = useI18N()
+    const { classes } = useStyles()
+    const [searchTerm, setSearchTerm] = useState<string>('')
+
+    const [market, setMarket] = useState<number>(0)
+    const [league, setLeague] = useState<number>(0)
+    const [sort, setSort] = useState<string>(DEFAULT_LABEL)
+
+    const { value, error, loading, retry } = useFetchApi(searchTerm, market, league, sort)
+
+    const events = value?.events
+    const eventsFiltered = value?.eventsFiltered
+
+    const sports = useSportList(events)
+    const leagues = useLeagueList(events)
+    const markets = useMarketList(events)
+    const sorts = [DEFAULT_LABEL, t.plugin_start_date(), t.plugin_newest()]
+
+    const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const term = event.target.value
+        setSearchTerm(term)
+    }
+
+    const marketSelect = usePortalShadowRoot((container: any) => (
+        <Select
+            MenuProps={{
+                container,
+                anchorOrigin: { horizontal: 'left', vertical: 'bottom' },
+                classes: { paper: classes.menuPaper },
+            }}
+            value={market.toString()}
+            label={t.plugin_market()}
+            onChange={(event: SelectChangeEvent) => setMarket(Number(event.target.value))}>
+            {markets?.map((market) => (
+                <MenuItem key={market.id} value={market.id}>
+                    {market.label}
+                </MenuItem>
+            ))}
+        </Select>
+    ))
+
+    const leagueSelect = usePortalShadowRoot((container) => (
+        <Select
+            MenuProps={{
+                container,
+                anchorOrigin: { horizontal: 'left', vertical: 'bottom' },
+                classes: { paper: classes.menuPaper },
+            }}
+            value={league.toString()}
+            label={t.plugin_league()}
+            onChange={(event: SelectChangeEvent) => setLeague(Number(event.target.value))}>
+            {leagues?.map((league) => (
+                <MenuItem key={league.id} value={league.id}>
+                    {league.label}
+                </MenuItem>
+            ))}
+        </Select>
+    ))
+
+    const sortSelect = usePortalShadowRoot((container) => (
+        <Select
+            MenuProps={{
+                container,
+                anchorOrigin: { horizontal: 'left', vertical: 'bottom' },
+                classes: { paper: classes.menuPaper },
+            }}
+            value={sort}
+            label={t.plugin_sort()}
+            onChange={(event: SelectChangeEvent) => setSort(event.target.value)}>
+            {sorts?.map((sort) => (
+                <MenuItem key={sort} value={sort}>
+                    {sort}
```

Comment:
```suggestion
                    <MenuText>{sort}</MenuText>
```

Reply:
MenuText does not exists. Plus, MenuItem is enough, is a li